datalogtool/src/main/native/cpp/App.cpp

```diff
@@ -110,6 +119,12 @@ static void DisplayMainMenu() {
     }
     ImGui::EndPopup();
   }
+
+  if (docs) {
+    wpi::gui::OpenURL(
+        "https://docs.wpilib.org/en/stable/docs/software/telemetry/"
+        "datalog.html");
+  }
```

Comment:
This can be simplified--you can call OpenURL directly from within the MenuItem check without using a boolean.  The only reason for the boolean approach is when you create popups, you don't want to start building the popup from within the menu.